Resources/Prototypes/_Mono/Shipyard/jug.yml

```
@@ -0,0 +1,53 @@
+  stations:
+    Jug:
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Jug {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)